projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -133,6 +135,7 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
   @Input() scrollbarV: boolean;
   @Input() scrollbarH: boolean;
   @Input() loadingIndicator: boolean;
+  @Input() customLoadingIndicator: boolean;
```

Comment:
Can we get rid of this boolean input.

I would this it should work as this.

- use the table as you are used too
- set loadingIndicator to true to see the progress indicator as you are used too
- Add the component <my-loading-indicator loading-indicator></my-loading-indicator> to the table and you see my loading indicator instead of the default one

I think this should be all from a user point of view. no additional APIs to be maintained and no breaking change to the original datatable